src/components/FormHelpMessage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View, ViewStyle, StyleProp} from 'react-native';
+import isEmpty from 'lodash/isEmpty';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+import themeColors from '../styles/themes/default';
+import styles from '../styles/styles';
+import * as Localize from '../libs/Localize';
+
+type FormHelpMessageProps = {
+    /** Error or hint text. Ignored when children is not empty */
+    message?: string | Array<string | Record<string, string>>;
+
+    /** Children to render next to dot indicator */
+    children?: React.ReactNode;
+
+    /** Indicates whether to show error or hint */
+    isError?: boolean;
+
+    /** Container style props */
+    style?: Array<StyleProp<ViewStyle>>;
+};
+
+function FormHelpMessage({message = '', children, isError = true, style = []}: FormHelpMessageProps) {
+    if (isEmpty(message) && isEmpty(children)) {
+        return null;
+    }
+
+    const translatedMessage = Localize.translateIfPhraseKey(message);
+    return (
+        <View style={[styles.flexRow, styles.alignItemsCenter, styles.mt2, styles.mb1, ...style]}>
```

Comment:
@pasyukevich @fabioh8010 Should we make the style be of type `StyleProp<ViewStyle>` and remove the spread operator here?

```
/** Container style props */
style?:  StyleProp<ViewStyle>;

....

<View style={[styles.flexRow, styles.alignItemsCenter, styles.mt2, styles.mb1, style]}>
```

Reply:
@VickyStash good catch! We can do that

Reply:
updated